pandas/core/indexes/datetimelike.py

```diff
@@ -286,7 +286,7 @@ def min(self, axis=None, *args, **kwargs):
         except ValueError:
             return self._na_value
 
-    def argmin(self, axis=None, *args, **kwargs):
+    def argmin(self, axis=None, skipna=True, *args, **kwargs):
```

Comment:
can these inherit the doc-string?

Reply:
I'll take a look.  Might also get rid of `*args` while at it

Reply:
Hmm the docstrings here and for the corresponding methods in base.IndexOpsMixin are kind of clunky.  This may merit a separate look, @datapythonista ?

Reply:
I don't know well what's the class hierarchy and whether makes sense to inherit. But we'll have to add `Parameters`, `Returns` and `Examples` section here if this is public.